src/NServiceBus.Core.Tests/AssemblyScanner/AssemblyScannerTests.cs

```diff
@@ -267,6 +325,12 @@ public DynamicAssembly(string nameWithoutExtension, DynamicAssembly[] references
                     param.ReferencedAssemblies.Add(reference.FilePath);
                 }
 
+                if (executable)
```

Comment:
Agreed. I don't see a problem with including this. There is a small chance that we could introduce something that is affected by the existence of this class/method and not having it would mean we would never catch that problem.